core/locale/module.go

```diff
@@ -34,11 +34,13 @@ func (m *Module) DefaultConfig() config.Map {
 		"locale": config.Map{
 			"locale": "en-US",
 			"accounting": config.Map{
-				"decimal":    ",",
-				"thousand":   ".",
-				"formatZero": "%s -,-",
-				"format":     "%s %v",
-				"formatLong": "%v %v",
+				"default": config.Map{
+					"decimal":    ",",
+					"thousand":   ".",
+					"formatZero": "%s -,-",
```

Comment:
Should this be `%s.-` instead of `%s -,-`?

Reply:
adjusted